src/components/inputs/description-field.tsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { Box, Button } from '@mui/material';
+import AddIcon from '@mui/icons-material/Add';
+import DeleteIcon from '@mui/icons-material/Delete';
+import { useFormContext } from 'react-hook-form'; // Import useFormContext
+import FieldConstants from '../../utils/field-constants';
+import ExpandingTextField from './react-hook-form/ExpandingTextField';
+
+function DescriptionField() {
+    const [isDescriptionFieldVisible, setDescriptionFieldVisibility] = useState(false);
+    const { setValue } = useFormContext();
+    const handleToggle = () => {
+        if (isDescriptionFieldVisible) {
+            setValue(FieldConstants.DESCRIPTION, '');
+        }
+        setDescriptionFieldVisibility((prev) => !prev);
+    };
```

Comment:
to reduce complexity, make a "handleOpenDescription" and a "handleCloseDescription" methods
we don't really need a toggle here, since there are two different buttons

Reply:
done